core/templates/filters/translate.pipe.spec.ts

```diff
@@ -52,8 +52,8 @@ class MockTranslateService {
 
     if (interpolateParams) {
       return str.replace(/\<\[\s?([^{}\s]*)\s?\]\>/g,
-        (substring: string, b: string) => {
-          let r = interpolateParams[b];
+        (substring: string, interpolateParamsKey: string) => {
+          let r = interpolateParams[interpolateParamsKey];
```

Comment:
Please use a better variable name than "r". I asked for this in the previous review too...

Reply:
Changed, `r` was not necessary here.